modules/react-components/src/components/input/url-input.tsx

```diff
@@ -112,6 +112,42 @@ export interface URLInputPropsInterface extends IdentifiableComponentInterface,
     skipInternalValidation?: boolean;
     isCustom?: boolean;
     addOriginByDefault?: boolean;
+    /**
+     * Resolve i18n tag for popupHeaderPositive content
+     */
+    popupHeaderPositive?: string;
```

Comment:
Lets use `ReactNode` as a type for scalability.

Reply:
This change will affect the other components.